GliaWidgets/Coordinator/RootCoordinator.swift

```diff
@@ -38,10 +38,7 @@ class RootCoordinator: SubFlowCoordinator, FlowCoordinator {
 
     private let navigationController = NavigationController()
     private let navigationPresenter: NavigationPresenter
-    private let presentingWindow = UIApplication.shared.windows.filter { $0.isKeyWindow }.first
-    private var presentingViewController: UIViewController? {
-        return presentingWindow?.topMostViewController()
-    }
+    private let gliaPresenter = GliaPresenter()
```

Comment:
imho, the property name could be `presenter` since it has the type `GliaPresenter`.

Reply:
Seemed more natural just below the `navigationPresenter`. Tbh I'd leave it as is if that doesn't bother you too much 😅